src/Pagination/components/PageButton.ts

```diff
@@ -24,38 +25,46 @@ export const PageButton: SFC<PageButtonProps> = (props) => {
 
         cssClass = "btn mx-button mx-name-paging-first";
         iconClass = "glyphicon glyphicon-step-backward";
+        ariaLabel = "Go to first page";
 
     } else if (props.buttonType === "previousButton") {
 
         cssClass = "btn mx-button mx-name-paging-previous";
         iconClass = "glyphicon glyphicon-backward";
+        ariaLabel = "Go to previous page";
 
     } else if (props.buttonType === "nextButton") {
 
         cssClass = "btn mx-button mx-name-paging-next";
         iconClass = "glyphicon glyphicon-forward";
+        ariaLabel = "Go to next page";
 
     } else if (props.buttonType === "lastButton") {
 
         cssClass = "btn mx-button mx-name-paging-last";
         iconClass = "glyphicon glyphicon-step-forward";
+        ariaLabel = "Go to last page";
     }
 
     if (props.showIcon === "default") {
         if (!props.buttonCaption) {
             return createElement("button", {
                     className: classNames(cssClass, disabledClass),
+                    disabled: props.isDisabled,
                     onClick,
-                    key: props.key
+                    key: props.key,
+                    "aria-label": ariaLabel
```

Comment:
If it will also help users that don't use assistive technology then we should use `title` instead `aria-label`. They have the same effect and they will work with both public.

Reply:
True, using title is way better! Changed it.

Reply:
Add aria-label again, because screen reader doesn't read out title when reading out the page. So now there's an aria-label and title.